hooks/new

```diff
@@ -11,17 +11,28 @@ prompt_for external_domain line \
   --default ''
 
 isoauth= # assigned below with prompt_for
-prompt_for isoauth boolean \
-	'Would you like to authenticate against an OAuth2 endpoint (Github / UAA)?'
+prompt_for isoauth boolean -i --default 'false' \
+	'Would you like to authenticate against an OAuth2 endpoint (Github / UAA)? [y|N]'
+
+secure=
+prompt_for secure boolean -i  --default  'true' \
+  'Would you like to secure the admin user with a generated password and optional username? [Y|n]'
+
+username='admin'
+if $secure ; then
+  prompt_for username line -i --default $username \
+    'Admin username:'
+fi
 
 (
 echo "---"
 echo "kit:"
 echo "  name:    $GENESIS_KIT_NAME"
 echo "  version: $GENESIS_KIT_VERSION"
-if [[ $isoauth == 'true' ]]; then
+if $isoauth || $secure; then
```

Comment:
Does secure automatically pull in oauth and the reverse? I'm wondering if these should be separate. You might want to use secure if not using oauth?

Reply:
That code is simply to say: "If we have one of the features, we need the "feature:" header, otherwise don't echo it.  They are in no other way related.

Reply:
Ahh ok thx for the clarification.